app/routes/events/view/tickets/access.js

```diff
@@ -0,0 +1,27 @@
+import Ember from 'ember';
+
+const { Route } = Ember;
+
+export default Route.extend({
+  titleToken() {
+    return this.i18n.t('Access codes');
+  },
+
+  model() {
+    return [
+      {
+        'accessCode'       : '#386',
+        'accessCodeUrl'    : 'https://github.com',
+        'accessibleTicket' : 'Early bird',
+        'validity'         : '3 months',
+        'status'           : true
```

Comment:
What is this status ? just a status `true` doesn't really make any sense does it ?.
Should be called `isActive`